examples/Deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb

```diff
@@ -106,6 +106,16 @@
     "### Import required libraries and functions"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install tensorflow
</pre><p>Let's move <code>%pip install tensorflow "feast&lt;0.20" faiss-gpu</code>  at the top of the first nb,  because we save the user features, item features and configuration files (feature views) into the feature repo folder.. we need that folder to be initialized.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Yeah, good catch, we do need `feast` in that notebook. Faiss is mentioned but not used as far as I can tell, but I suppose it doesn't hurt to install it.